### PR TITLE
New Administrator Interface Layout

### LINKED ISSUE #77: When a route is removed, the stops that were associated with that route become inaccessable.

### PR DESCRIPTION
**Pull request by jlyon1**
Fixes #77 #72

### CODE REVIEW

File: static/js/vehicles.js [hunk 1]
```diff
@@ -1,25 +1,18 @@
...
     <div class = "button" @click="send" style="width: 50px;">add</div>
-
     </div>`,
     data (){
       return{
 
         ID: "",
         name: "",
-        active: true
+        // active: true,
```
Comment (kochman) [inline static/js/vehicles.js]: This can get nuked I think?

File: static/js/stops.js [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+            {
+              draggable: true
+            });
+          el.addStopMarker.addTo(el.StopsMap);
+
+        });
+        // this.StopsMap.on('click', function(e) {
+        //   el.RoutingWaypoints.push(e.latlng);
+        //   el.RoutingControl.setWaypoints(el.RoutingWaypoints);
+        // });
```
Comment (kochman) [inline static/js/stops.js]: Delete?

File: static/js/stops.js [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+          complete: function(data){
+            refresh=true;
+
+          }
+        });
+      },
+      getJSON: function(){
+        var el = this;
+        console.log(this.addStopMarker.getLatLng().lat);
+        //coords = this.drawnRoute.getLatLngs();
```
Comment (kochman) [inline static/js/stops.js]: Remove this and the preceding `console.log`?

File: static/js/stops.js [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+  data (){
+    return{
+      stopData: [{shuttleID:22}],
+      routeIDS: [],
+    };
+  },
+  mounted(){
+    var el = this;
+    $.get("/stops",function(data){
+      //console.log(data);
```
Comment (kochman) [inline static/js/stops.js]: Delete?

File: static/js/routes.js [hunk 5]
```diff
@@ -0,0 +1,247 @@
...
+        var toSend = {
+          "name":this.name,
+          "description":this.description,
+          "startTime":"",
+          "endTime":"",
+          "enabled":this.enabled.toString(),
+          "color":this.color,
+          "width":this.width.toString(),
+          "coords":JSON.stringify(coords)};
+          // console.log(toSend)
```
Comment (kochman) [inline static/js/routes.js]: Nuke?

File: static/js/routes.js [hunk 6]
```diff
@@ -0,0 +1,247 @@
...
+    </div>`,
+    data (){
+      return{
+
+      };
+    },
+    methods: {
+      submitForm: function(){
+        toSend = $("#jsontxt").val();
+        // console.log("val" + toSend)
```
Comment (kochman) [inline static/js/routes.js]: Obliterate?

**Comment by kochman** [comment]
- After adding a vehicle, name field is cleared, but not ID field
- Small width screens have text falling off the left tabs
- Adding and immediately removing a vehicle doesn't work
- Sometimes button titles are capitalized, sometimes not. Should always be "Sentence case"

**Comment by jlyon1** [comment]
Changing the route a stop is assigned to creates a duplicate stop.

**Comment by jlyon1** [comment]
Made those changes @kochman

**Comment by kochman** [comment]
This is very nice @jlyon1. Only thing I saw is adding and immediately removing a vehicle doesn't work (e.g. ID "hi" and name "hi") but I think we can track that in a separate issue (#110).

**Pull request merged**